specs/disciplina/educator/api/student.yaml

```diff
@@ -0,0 +1,414 @@
+      operationId: deleteSubmission
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/SubmissionHashPath'
+      responses:
+        '200':
+          description: successful operation
+        '403':
+          description: Cannot delete a graded submission
+        '404':
+          description: Submission with given hash not found
+    
+  /proofs:
+    get:
+      tags:
+        - proofs
+      summary: Get available proofs for student
+      description: Gets private transactions together with corresponding Merkle subtrees. Transactions from same blocks are grouped together and each group has one proof, which is a corresponding Merkle subtree.
+      operationId: getProofs
+      produces:
+        - application/json
+      parameters:
+        - in: query
+          name: since
+          description: Do not include transactions which were created earlier this time
+          required: false
+          type: string
+          format: date-time
+      responses:
+        200:
+          description: successfull operation
+          schema:
+            $ref: '#/definitions/Proofs'
+
+#securityDefinitions:
+  # api_key:
+  #   type: apiKey
+  #   name: api_key
+  #   in: header
+  
+definitions:
+  Hash:
+    type: string
+    format: byte
+    
+  Address:
+    type: string
+    format: byte
+    
+  Time:
+    type: string
+    format: date-time
+    
+  Subject:
+    type: integer
+    format: int32
+    
+  CourseId:
+    type: integer
+    format: int32
+    
+  Course:
+    type: object
+    required:
+      - id
+    properties:
+      id:
+        $ref: '#/definitions/CourseId'
+      desc:
+        type: string
+      subjects:
+        type: array
+        items:
+          $ref: '#/definitions/Subject'
+      is_enrolled:
+        type: boolean
+        
+  Assignment:
+    type: object
+    required:
+      - hash
+      - course_id
+      - contents_hash
+      - is_final
+      - desc
+    properties:
+      hash:
+        $ref: '#/definitions/Hash'
+      course_id:
+        $ref: '#/definitions/CourseId'
+      contents_hash:
+        $ref: '#/definitions/Hash'
+      is_final:
+        type: boolean
+      desc:
+        type: string
+      last_submission:
+        $ref: '#/definitions/Submission'
+        
+  Submission:
+    type: object
+    required:
+      - hash
+      - contents_hash
+      - assignment_hash
+    properties:
+      hash:
+        $ref: '#/definitions/Hash'
+      contents_hash:
+        $ref: '#/definitions/Hash'
+      assignment_hash:
+        $ref: '#/definitions/Hash'
+      grade:
+        $ref: '#/definitions/Grade'
+        
+  Grade:
+    type: object
+    required:
+      - grade
+      - timestamp
+      - has_proof
+    properties:
+      grade:
+        type: integer
+        format: int32
+      timestamp:
+        $ref: '#/definitions/Time'
+      has_proof:
```

Comment:
What does this field mean?

Reply:
Ok, so technically it's `was_published`.

Reply:
@volhovm That would be incorrect, because the _grade_ isn't _published_. `has_proof` is way less confusing, I believe.